documentation/sphinx/source/release-notes/release-notes-730.rst

```diff
@@ -2,6 +2,21 @@
 Release Notes
 #############
 
+7.3.57
+======
+* Same as 7.3.56 release with AVX enabled.
+
+7.3.56
+======
+* Manual flush if the rocksdb flush does not happen within a time interval `(PR #11792) <https://github.com/apple/foundationdb/pull/11792>`_
+* Have RocksDB histogram sample rate updated to 1 `(PR #11794) <https://github.com/apple/foundationdb/pull/11794>`_
+* TLS should accept same key with different values. `(PR #11763) <https://github.com/apple/foundationdb/pull/11763>`_
```

Comment:
@kakaiu what's the user impact for this PR? We need to state what problem this PR fixed here.

Reply:
This PR solves a problem of using TLS. Sometimes, TLS input string can contain same NID with multiple different Criteria. The existing implementation uses std::map<NID, Criteria> with insert operation which can only store the left-most NID-Criteria pair in the TLS input string. This PR allows to use same NID with multiple different Criteria  as the user input string by using std::set<std::pair<NID, Criteria>> to store the Criteria.